languages/python/exercises/concept/currency-exchange/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+## Numbers
+
+There are three different kinds of built-in numbers: integers, floating-point, and complex. In this exercise you will only be dealing with ints and floats.
+
+### ints
+
+`ints` are whole numbers. e.g. `1234`, `-10`, `20201278`.
+
+### floats
+
+`floats` are numbers containing a decimal point. e.g. `0.0`,`3.14`,`-9.01`.
+
+You can see more detail the following resources:
+
+- [Python numeric type documentation](https://docs.python.org/3/library/stdtypes.html#typesnumeric)
+- [Documentation for `int()` built in](https://docs.python.org/3/library/functions.html#int)
+- [Documentation for `float()` built in](https://docs.python.org/3/library/functions.html#float)
```

Comment:
```suggestion
- [Python numeric type documentation][numeric-type-docs]
- [Documentation for `int()` built in][`int()` built in]
- [Documentation for `float()` built in][`float()` built in]
```

Reply:
For some reason, I am unable to add the reflinks to the bottom of this file a suggestion...so I am going to list them in this comment.  The reflinks below need to be added to the bottom of this document.  This is to replace the in-document links, which are harder to read and maintain.

[arbitrary-precision]: https://en.wikipedia.org/wiki/Arbitrary-precision_arithmetic#:~:text=In%20computer%20science%2C%20arbitrary%2Dprecision,memory%20of%20the%20host%20system.
[numeric-type-docs]: https://docs.python.org/3/library/stdtypes.html#typesnumeric
[`int()` built in]: https://docs.python.org/3/library/functions.html#int
[`float()` built in]: https://docs.python.org/3/library/functions.html#float